src/components/singleRocket.css

```diff
@@ -0,0 +1,60 @@
+.rocket-section {
+  display: flex;
+  padding: 8px;
+  height: 200px;
+}
+
+.rocket-info {
+  display: flex;
+}
+
+.rocket-image {
+  width: 16%;
+}
+
+.rocket-detail {
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 8px;
+}
+
+.rocket-summary {
+  text-align: initial;
+}
+
+.rocket-name {
+  display: contents;
+}
+
+.cancel-reservation,
+.reserve-rocket {
+  border-radius: 5px;
+  padding: 0.5rem;
+}
+
+.cancel-reservation {
+  font-weight: 600;
+  width: 10rem;
+}
+
+.reserve-rocket {
+  background-color: #1375cc;
+  border: none;
+  box-shadow: 6px 6px 10px -1px rgb(0 0 0 / 15%), -6px -6px 10px -1px hsl(0deg 0% 100% / 70%);
+  color: #fff;
+  width: 8rem;
+}
```

Comment:
- [OPTIONAL] I think it will be better if you remove the width of your buttons, all both are too big 🙃 😉

Reply:
Thank you @cynthiainga for your suggestion, I will surely implement this change 😄